components/drafts/draft_actions/delete_draft_modal.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import GenericModal from 'components/generic_modal';
+
+type Props = {
+    displayName: string;
+    onConfirm: () => void;
+    onExited: () => void;
+}
+
+function DeleteDraftModal({
+    displayName,
+    onExited,
+    onConfirm,
+}: Props) {
+    const {formatMessage} = useIntl();
+
+    const title = formatMessage({
+        id: 'drafts.confirm.delete.title',
+        defaultMessage: 'Delete Draft',
+    });
+
+    const confirmButtonText = formatMessage({
+        id: 'drafts.confirm.delete.button',
+        defaultMessage: 'Yes, Delete',
+    });
+
+    const message = (
+        <FormattedMessage
+            id={'drafts.confirm.delete.text'}
+            defaultMessage={'Are you sure you want to delete this draft to <strong>{displayName}</strong>?'}
+            values={{
+                strong: (chunk: string) => <strong>{chunk}</strong>,
+                displayName,
+            }}
+        />
+    );
+
+    return (
+        <GenericModal
+            confirmButtonClassName='delete'
+            confirmButtonText={confirmButtonText}
+            handleCancel={() => {}}
+            handleConfirm={onConfirm}
+            modalHeaderText={title}
+            onExited={onExited}
+        >
+            <div>{message}</div>
```

Comment:
Do we need block level element? I think we can get rid of div. or maybe 

```typescript
<FormattedMessage
            id={'drafts.confirm.delete.text'}
             tagName={'div'}
```
This may apply to some other places as well.

Reply:
Yeah just tested it and doesn't look like we need it :+1: